core/trino-main/src/test/java/io/trino/server/ui/TestWebUi.java

```diff
@@ -761,7 +761,7 @@ private void assertAuth2Authentication(HttpServerInfo httpServerInfo, String acc
         assertEquals(cookie.getValue(), accessToken);
         assertEquals(cookie.getPath(), "/ui/");
         assertEquals(cookie.getDomain(), baseUri.getHost());
-        assertTrue(cookie.getMaxAge() > 0 && cookie.getMaxAge() < MINUTES.toSeconds(5));
+        assertThat(cookie.getMaxAge()).isGreaterThan(0).isLessThan(MINUTES.toSeconds(1));
```

Comment:
You are deleting this line in the next commit, is it worth the effort?

Reply:
When I start working on this I spotted this so I made a commit.
This line is moved but as you said this change is so small so I will squash the commits